### ISSUE TITLE
Missing instructions for Austin

### ISSUE THREAD

**Issue by P403n1x87** [jlfwong/speedscope#302]
[Austin](https://github.com/P403n1x87/austin) is a fast frame stack sampler for CPython. The output produced by it is in the collapsed stack format, which is now supported natively by speedscope. Hence it is enough to dump the data collected by Austin into a file and upload it on speedscope.app for a quick visualisation

~~~
austin -o my_profile_data.aprof python3 myscript.py
~~~

The repository includes sample Python code with an conversion utility that converts Austin output into the speedscope JSON format. This is handy when profiling in full mode so that both time and memory profiles are generated

~~~
austin -fo my_full_profile_data.aprof python3 myscript.py
austin2speedscope my_full_profile_data.aprof my_full_profile_data.json
~~~

**Comment by jlfwong** [comment]
Hi @P403n1x87! I'd accept a simple PR to update the README to link to https://github.com/P403n1x87/austin or https://github.com/P403n1x87/austin#speedscope

**Comment by P403n1x87** [comment]
Resolved by  #303.
